docs/extended_api/shapes/aligned_size_t.md

```diff
@@ -20,6 +20,11 @@ struct cuda::aligned_size_t {
 The class template `cuda::aligned_size_t` is a _shape_ representing an extent
   of bytes with a statically defined (address and size) alignment.
 
+*Preconditions*: 
+
+- The _address_ of the extent of bytes must be aligned to an `Alignment` alignment boundary.
+- The _size_ of the extent of bytes in bytes must be a multiple of the `Alignment`.
```

Comment:
Suggest simplifying to: ...extent of bytes ~~in bytes~~ must be...

There hasn't been a point where we've been unequivocal about whether or not the size parameter is operating on anything other than a count of bytes.

I don't think we need to restructure this to make the point clear. I'm happy with the language.

Reply:
(resolved) I just wanted to be clear that the size parameter operates in bytes. There are some `cooperative_groups::memcpy_async(T*, T*, Shape)` overloads where the `size_t` shape operates in terms of element counts, while the `aligned_size_t` shape operates in terms of bytes, which is a bit confusing TBH).